tests/storage/guestfs.go

```diff
@@ -208,7 +208,7 @@ var _ = SIGDescribe("[rfe_id:6364][[Serial]Guestfs", func() {
 		It("Should successfully run guestfs command on a filesystem-based PVC with root", func() {
 			f := createFakeAttacher()
 			defer f.closeChannel()
-			pvcClaim = "pvc-fs-with-different-uid"
+			pvcClaim = "pvc-fs-with-root"
```

Comment:
I think you either have to add an AfterEach to ensure that he pod is terminated, or randomize the pvc and pod names to avoid conflicts.

Reply:
Or do the other tests already user different names?

Reply:
Yes, every test should use a different claim name. I did a wrong copy paste and 2 tests have the same PVC name. See https://github.com/kubevirt/kubevirt/blob/9d80854a1c8ec671bd17e2e81db69da1a804a1f0/tests/storage/guestfs.go#L189 and https://github.com/kubevirt/kubevirt/blob/main/tests/storage/guestfs.go#L211 .
@maya-r fix is correct

Reply:
Perfect.